arch/platform/cooja/dev/cooja-radio.c

```diff
@@ -54,6 +54,8 @@
 #define COOJA_RADIO_BUFSIZE 125
 #endif
 
+#define MIN_CHANNEL 15
+#define MAX_CHANNEL 26
```

Comment:
Instead of starting on channel 15, what do you think about using channels 11 - 26 to match IEEE 802.15.4 2.4GHz radio?